Add FTPS support

daradib opened the pull request:
Support FTPS (TLS) on Python 2.7/3.2+ where ftplib.FTP_TLS exists.
Also change examples to show ftps:// but retain non-TLS default
for backwards-compatibility.

Would like to use pyftpsync with [Lektor](https://github.com/lektor/lektor).

mar10 commented:
Good work, thanks for contributing!
Your PR contains a lot of whitespace changes, that makes it hard to see the important parts, so I cleaned that up in the master. Alas, that seemed to create 2-3 minor conflicts. Do you think you can update your PR from master, to fix that?

I also had to drop Py2.6 and 3.2, since support is dropped in tox/travis, so your PR should pass the checks.

I saw that you changed the tests, dependent on the python version. 
However I think it would be better to keep the ftp: tests, and add ftps: on top.

daradib commented:
@mar10 Thanks for the feedback and thanks for waiting. I've rebased on to master and removed the remaining trailing whitespace deletions. I kept tests for both ftp and ftps.
